.gitignore

```diff
@@ -9,3 +9,4 @@ build
 .idea
 out
 
+/bin/
```

Comment:
After coming back to the project a few days later, I pulled it back into Eclpise.  It's Eclipse that is adding the /bin folder. I can see .idea is ignored already so it might actually make sense to ignore /bin. If i ever issue a PR again - I'll likely add this to the gitignore.